packages/react-scripts/template/app/server/lib/assets.js

```diff
@@ -1,7 +1,25 @@
 var { isDev } = require('../system');
 
+// @todo - should these functions be moved to react-scripts
```

Comment:
Say react-scripts is import in the template like:

`var { getManifest } = require('@lostmyname/react-scipts');`

how will this work in development?

One option would be to run the node code through webpack and add an alias for @lostmyname/react-scripts that points to the local package.